src/libraries/System.Drawing.Common/src/System/Drawing/Internal/GPStream.COMWrappers.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Runtime.InteropServices;
+
+namespace System.Drawing.Internal
+{
+    internal sealed partial class GPStream : Interop.Ole32.IStream
+    {
+        public unsafe Interop.HRESULT Clone(IntPtr* ppstm)
```

Comment:
Do the gdi methods actually call Clone or CopyTo? It is fairly common to leave these methods unimplemented on IStream.

Reply:
E.g. https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/coreclr/utilcode/fstream.cpp#L270

Reply:
Using the unit tests, I haven't seen `Clone` being in called. But I do see `CopyTo` being invoked. For example:

```
>	System.Drawing.Common.dll!System.Drawing.Internal.GPStream.CopyTo(System.IntPtr pstm, ulong cb, ulong* pcbRead, ulong* pcbWritten) Line 31	C#
 	System.Drawing.Common.dll!System.Drawing.DrawingComWrappers.IStreamVtbl.CopyTo(System.IntPtr thisPtr, System.IntPtr pstm, ulong cb, ulong* pcbRead, ulong* pcbWritten) Line 177	C#
 	[Native to Managed Transition]	
 	GdiPlus.dll!GetEmf()	Unknown
 	GdiPlus.dll!GetHeaderAndMetafile()	Unknown
 	GdiPlus.dll!GpMetafile::InitStream()	Unknown
 	GdiPlus.dll!GpMetafile::GpMetafile(struct IStream *,enum WmfCompatibilityMode)	Unknown
 	GdiPlus.dll!GdipCreateMetafileFromStream()	Unknown
 	[Managed to Native Transition]	
 	System.Drawing.Common.dll!System.Drawing.Imaging.Metafile.CreateGdipMetafileFromStream(System.Drawing.Internal.GPStream stream) Line 18	C#
 	System.Drawing.Common.dll!System.Drawing.Imaging.Metafile.Metafile(System.IO.Stream stream) Line 37	C#
 	System.Drawing.Common.Tests.dll!MonoTests.System.Drawing.Imaging.MetafileTest.GetMetafileHeader_FromMemoryStream_WmfPlaceable() Line 138	C#
```